src/mlpack/tests/lsh_test.cpp

```diff
@@ -132,8 +132,8 @@ BOOST_AUTO_TEST_CASE(NumTablesTest)
     fail = false;
 
     const int lSize = 6; // Number of runs.
-    const int lValue[] = {1, 8, 16, 32, 64, 128}; // Number of tables.
-    double lValueRecall[lSize] = {0.0}; // Recall of each LSH run.
+    const int lValue[] = { 1, 8, 16, 32, 64, 128 }; // Number of tables.
```

Comment:
@zoq Are we following this spacing style for non-arma matrices

Reply:
Hm, I think we don't have a style guide for such cases, maybe we do?

Reply:
I do not know either, I usually put spaces, but my style is different from the one used in mlpack, so I am checking